Test failures since 0.25.0

Hi, I noticed that a couple of unit tests started to fail since 0.25.0 release, could you check if that's a regression? I'm using the archive from pypi.

```
=================================== FAILURES ===================================
____________________________ test_lint_free_pylint _____________________________

    def test_lint_free_pylint():
        # Can't use temp_document because it might give us a file that doesn't
        # match pylint's naming requirements. We should be keeping this file clean
        # though, so it works for a test of an empty lint.
>       assert not pylint_lint.pyls_lint(
            Document(uris.from_fs_path(__file__)), True)
E       AssertionError: assert not [{'code': 'C0111', 'message': '[missing-docstring] Missing module docstring', 'range': {'end': {'character': 29, 'line...ing', 'range': {'end': {'character': 29, 'line': 61}, 'start': {'character': 0, 'line': 61}}, 'severity': 3, ...}, ...]
E        +  where [{'code': 'C0111', 'message': '[missing-docstring] Missing module docstring', 'range': {'end': {'character': 29, 'line...ing', 'range': {'end': {'character': 29, 'line': 61}, 'start': {'character': 0, 'line': 61}}, 'severity': 3, ...}, ...] = <function pyls_lint at 0x7f32f24f9950>(<pyls.workspace.Document object at 0x7f32e37fc3c8>, True)
E        +    where <function pyls_lint at 0x7f32f24f9950> = pylint_lint.pyls_lint
E        +    and   <pyls.workspace.Document object at 0x7f32e37fc3c8> = Document('file:///build/python-language-server/src/python-language-server-0.25.0/test/plugins/test_pylint_lint.py')
E        +      where 'file:///build/python-language-server/src/python-language-server-0.25.0/test/plugins/test_pylint_lint.py' = <function from_fs_path at 0x7f32f2d1db70>('/build/python-language-server/src/python-language-server-0.25.0/test/plugins/test_pylint_lint.py')
E        +        where <function from_fs_path at 0x7f32f2d1db70> = uris.from_fs_path

test/plugins/test_pylint_lint.py:66: AssertionError
____________________________ test_per_file_caching _____________________________

    def test_per_file_caching():
        # Ensure that diagnostics are cached per-file.
        with temp_document(DOC) as doc:
            assert pylint_lint.pyls_lint(doc, True)
    
>       assert not pylint_lint.pyls_lint(
            Document(uris.from_fs_path(__file__)), False)
E       AssertionError: assert not [{'code': 'C0111', 'message': '[missing-docstring] Missing module docstring', 'range': {'end': {'character': 29, 'line...ing', 'range': {'end': {'character': 29, 'line': 61}, 'start': {'character': 0, 'line': 61}}, 'severity': 3, ...}, ...]
E        +  where [{'code': 'C0111', 'message': '[missing-docstring] Missing module docstring', 'range': {'end': {'character': 29, 'line...ing', 'range': {'end': {'character': 29, 'line': 61}, 'start': {'character': 0, 'line': 61}}, 'severity': 3, ...}, ...] = <function pyls_lint at 0x7f32f24f9950>(<pyls.workspace.Document object at 0x7f32f0274898>, False)
E        +    where <function pyls_lint at 0x7f32f24f9950> = pylint_lint.pyls_lint
E        +    and   <pyls.workspace.Document object at 0x7f32f0274898> = Document('file:///build/python-language-server/src/python-language-server-0.25.0/test/plugins/test_pylint_lint.py')
E        +      where 'file:///build/python-language-server/src/python-language-server-0.25.0/test/plugins/test_pylint_lint.py' = <function from_fs_path at 0x7f32f2d1db70>('/build/python-language-server/src/python-language-server-0.25.0/test/plugins/test_pylint_lint.py')
E        +        where <function from_fs_path at 0x7f32f2d1db70> = uris.from_fs_path

test/plugins/test_pylint_lint.py:103: AssertionError
```

I can even reproduce this locally:

```
pylint test/plugins/test_pylint_lint.py
************* Module test.plugins.test_pylint_lint
test/plugins/test_pylint_lint.py:1:0: C0111: Missing module docstring (missing-docstring)
test/plugins/test_pylint_lint.py:25:0: C0111: Missing function docstring (missing-docstring)
test/plugins/test_pylint_lint.py:36:0: C0111: Missing function docstring (missing-docstring)
test/plugins/test_pylint_lint.py:41:0: C0111: Missing function docstring (missing-docstring)
test/plugins/test_pylint_lint.py:52:0: C0111: Missing function docstring (missing-docstring)
test/plugins/test_pylint_lint.py:62:0: C0111: Missing function docstring (missing-docstring)
test/plugins/test_pylint_lint.py:70:0: C0111: Missing function docstring (missing-docstring)
test/plugins/test_pylint_lint.py:98:0: C0111: Missing function docstring (missing-docstring)

------------------------------------------------------------------
Your code has been rated at 8.33/10 (previous run: 8.33/10, +0.00)
```

I think you guys have some `~/.pylintrc` that disables `C` and that's why tests pass for you, but tests fail in an isolated environment 😉 

I'm really not sure how come your CI passes, have you pre-configured it somehow with a custom `pylintrc`?

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -2,4 +2,5 @@ include README.rst
 include versioneer.py
 include pyls/_version.py
 include LICENSE
+include .pylintrc
 recursive-include test *.py

